s3torchbenchmarking/src/s3torchbenchmarking/benchmark.py

```diff
@@ -71,6 +74,39 @@ def from_conf(dataset_config: DictConfig):
             return DatasetSharding[dataset_config.sharding]
 
 
+def parse_s3_url(url: str) -> (str, str):
```

Comment:
This is implemented in s3torchconnector. Why not import it? (I think a private import here is probably OK as it's the same repo)